pkgs/by-name/ty/typst/package.nix

```diff
@@ -46,13 +47,23 @@ rustPlatform.buildRustPackage rec {
     OPENSSL_NO_VENDOR = true;
   };
 
+  # Fix for "Found argument '--test-threads' which wasn't expected, or isn't valid in this context"
+  postPatch = ''
```

Comment:
Is this something we can submit upstream ?

Reply:
Hmmm maybe. But I wonder if it'd be better if we fixed this in nixpkgs instead? It seems to be a quite frequent issue:

```
$ grep -r "Found argument '--test-threads'"
pkgs/applications/misc/navi/default.nix:    # error: Found argument '--test-threads' which wasn't expected, or isn't valid in this context
pkgs/applications/version-management/glitter/default.nix:  # error: Found argument '--test-threads' which wasn't expected, or isn't valid in this context
pkgs/servers/search/sonic-server/default.nix:  # Found argument '--test-threads' which wasn't expected, or isn't valid in this context
pkgs/tools/misc/lsd/default.nix:  # Found argument '--test-threads' which wasn't expected, or isn't valid in this context
pkgs/tools/misc/sfz/default.nix:  # error: Found argument '--test-threads' which wasn't expected, or isn't valid in this context
pkgs/tools/text/tidy-viewer/default.nix:  # error: Found argument '--test-threads' which wasn't expected, or isn't valid in this context
pkgs/by-name/ty/typst/package.nix:  # Fix for "Found argument '--test-threads' which wasn't expected, or isn't valid in this context"
```

There's apparently a [`RUST_TEST_THREADS` environment variable](https://manpages.debian.org/bullseye/rustc/rustc.1.en.html#RUST_TEST_THREADS), maybe we should use it instead?

Reply:
yeah let's try

Reply:
Here's a draft PR: https://github.com/NixOS/nixpkgs/pull/312505

This should target staging, right? As probably causes rebuild for every single rust package.

Reply:
Yes indeed.

Reply:
Let's merge this PR and refactor it when #312505 has landed.